CHANGELOG.md

```diff
@@ -1,6 +1,50 @@
 Change Log
 ===
 
+v4.1.0
+---
+* [new-rule] `prefer-const` rule with fixer added (#1801)
```

Comment:
I don't think you need "rule added" in all these places because the line already starts with [new rule]. just say the rule name and indicate if a fixer is available